routes/api/contacts.js

```diff
@@ -1,25 +1,85 @@
-const express = require('express')
+import express from "express"
+import contactsServices from '../../models/contacts.js';
+import { HttpError } from '../../helpers/index.js';
+import Joi from "joi";
 
 const router = express.Router()
 
+const contactScheme = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
```

Comment:
Schema must be defined in separate file schema.js. You should validate email and phone properly

Reply:
Hi, create file schema.js